### ISSUE TITLE
React: Update codeblock delimiters

### ISSUE THREAD

**Issue by aconstas** [TheOdinProject/curriculum#27459]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

Codeblocks should be updated to align with new changes in our [layout style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

In the sections named in this issue, codeblocks will be opened by a set of 3 tildes (the ~ character) and then closed by another set of 3 tildes. To align with the style guide, these tilde characters need to change to backticks (the ` character). Be sure to refer to [the codeblocks section](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#codeblocks) of the style guide for an example of what this should look like.
## Guidance:
- Comment on this issue to get a section assigned to you. You can also ask for a specific one
- Please wait until you are assigned by a maintainer before starting to work on this issue to avoid multiple people working on the same sections
- Make sure to read the [contributing guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md) before starting your first contribution
- Please add the following to your PR in the Issues section: Related to #xxxxx, since this issue is divided into sections, individual PRs should NOT INCLUDE the closes keyword.
- **Please ensure that you update the delimiters in every listed lesson of the section you are assigned to.**

### [Getting Started With React (4 lessons)](https://www.theodinproject.com/paths/full-stack-javascript/courses/react#getting-started-with-react)
- [x] @Churtified  `React Components`, `What is JSX?`, `Rendering Techniques`, `Passing Data Between Components`

### [States and Effects(3 lessons)](https://www.theodinproject.com/paths/full-stack-javascript/courses/react#states-and-effects)
- [x] @marlatte  `Introduction to State`, `More on State`, `How to Deal with Side Effects`

### [Class Components (2 lessons)](https://www.theodinproject.com/paths/full-stack-javascript/courses/react#class-components)
- [x] @roopxx `Class Based Components`, `Component Lifecycle Methods`

### [React Testing (2 lessons)](https://www.theodinproject.com/paths/full-stack-javascript/courses/react#react-testing)
- [x] @kamranzafar4343 `Introduction to React Testing`, `Mocking Callbacks and Components`

### [The React Ecosystem (3 lessons)](https://www.theodinproject.com/paths/full-stack-javascript/courses/react#the-react-ecosystem)
- [x] @ParagPalkar `Project Shopping Cart`, `React Router`, `Type Checking with Proptypes`

### [More React Concepts (3 lessons)](https://www.theodinproject.com/paths/full-stack-javascript/courses/react#more-react-concepts)
- [x] @Ervin001 `Managing State with Context API`, `Reducing State`, `Refs and Memoization`

@JoshDevHub

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/paths/full-stack-javascript/courses/react

**Comment by Churtified** [comment]
I can do a section, thanks!

**Comment by marlatte** [comment]
I'll take a section as well!

**Comment by roopxx** [comment]
I'll take one section too. Thanks!

**Comment by JoshDevHub** [comment]
Thank you for volunteering @roopxx 

I've assigned you the Class Components section.

**Comment by JoshDevHub** [comment]
Thank you for volunteering @kamranzafar4343 

I've assigned you the section on React testing.

**Comment by ParagPalkar** [comment]
I would like to take a lesson from one of these !
Thank You !

**Comment by Ervin001** [comment]
If there are any available I would like to take a section as well.

**Comment by MaoShizhong** [comment]
@ParagPalkar assigned you the section on `The React Ecosystem`

@Ervin001 assigned you the section on `More React Concepts`

**Comment by BryanF1nes** [comment]
If any of these still need work done, I'd be happy to help. :)

**Comment by JoshDevHub** [comment]
@BryanF1nes Thank you for volunteering.

All the items in this issue have been assigned. However, there are still a couple of more issues coming in the future for codeblock fixes. I'll be sure to include you in the next round.

**Comment by BryanF1nes** [comment]
@JoshDevHub Wonderful, thanks!

**Comment by MaoShizhong** [comment]
Thank you to everyone who contributed to this!
